ingestion/src/levels.rs

```diff
@@ -180,14 +177,16 @@ pub async fn fetch_levels(stinfo_conn_string: &str) -> Result<ParamLevelTable, E
         let default_hlevel = standard_hlevel.unwrap_or(0).abs();
 
         // convert the `sensorlevel_id` to a direction
-        // Down if `sensorlevel_id` from stinfosys contains the word "below"
+        // Down if `sensorlevel_id` from stinfosys is depth below surface or depth below sea surface
+        // Currently 3 values: "height_above_ground", "depth_below_surface", and "depth_below_sea_surface"
+        // need to change code if more are added to stinfosys
         let direction = match sensorlevel_id {
+            Some("height_above_ground") => Direction::Up,
+            Some("depth_below_surface") => Direction::Down,
+            Some("depth_below_sea_surface") => Direction::Down,
```

Comment:
Can be written maybe more neatly as

```rust
Some("depth_below_surface") | Some("depth_below_sea_surface") => Direction::Down
```

Up to you which version you prefer though